src/autoyast-rnc/ntpclient.rnc

```diff
@@ -4,77 +4,22 @@ namespace config = "http://www.suse.com/1.0/configns"
 
 ntp-client =
   element ntp-client {
-    configure_dhcp? &
-    peers? &
-    restricts? &
-    start_at_boot? &
-    start_in_chroot? &
     ntp_policy? &
-    sync_interval? &
-    synchronize_time?
+    ntp_sync? &
+    ntp_servers?
   }
 
 ntp_policy =
     element ntp_policy { text }
-create_symlink =
-  element create_symlink { BOOLEAN }
-configure_dhcp =
-  element configure_dhcp { BOOLEAN }
-# ignored since 2.16.8
-initial_sync =
-  element initial_sync { BOOLEAN }
-fudge_comment = element fudge_comment { text }
-fudge_options = element fudge_options { text }
-param = element param { text }
-peer =
-  element peer {
-    (peer_address
-     | peer_comment
-     | create_symlink
-     | peer_device
-     | fudge_comment
-     | fudge_options
-     | initial_sync
-     | peer_options
-     | param
-     | peer_type)*
-  }
-peer_type = element type { text }
-peer_options = element options { text }
-peer_device  = element device  { text }
-peer_comment = element comment { text }
-peer_address = element address { text }
-
-peers =
-  element peers {
-    LIST,
-    peer*
-  }
-
-restrict =
-  element restrict {
-    (restrict_mask
-     | restrict_comment
-     | restrict_options
-     | restrict_target)*
-  }
-restrict_mask = element mask { text }
-restrict_comment = element comment { text }
-restrict_options = element options  { text }
-restrict_target = element target { text }
-
-restricts =
-  element restricts {
-    LIST,
-    restrict*
-  }
-
-start_at_boot =
-  element start_at_boot { BOOLEAN }
-start_in_chroot =
-  element start_in_chroot { BOOLEAN }
+# possible values are systemd, manual and number,
```

Comment:
> Possible values are "systemd", "manual", or a number which means the interval in seconds between synchronizations

An integer or a float?

Reply:
good note, it is integer. ( float there is rounded )